src/app/self-sourced-project/self-sourced-project.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 
+
```

Comment:
When you commit your changes it is a good practice to go over all files being committed and review changes to make sure you don't introduce any unwanted changes. You can easily do this in VS Code (are you using VS Code?) in SOURCE CONTROL by double clicking files - it'll open comparison window. I wouldn't commit changes to this file. You can roll then back during review.

Reply:
Thanks, Alex, I found that feature!